filters/private/Segmentation.hpp

```diff
@@ -91,10 +91,68 @@ std::ostream& operator<<(std::ostream& out, const PointClasses& classes);
   \param[in] tolerance the tolerance for adding points to a cluster.
   \returns a vector of clusters (themselves vectors of PointIds).
```

Comment:
No longer vectors.

In this case, I can't imagine it much matters if you use deque or vector.